OPHD/UI/IconGrid.h

```diff
@@ -15,6 +15,10 @@
 
 #include <algorithm>
 
+#include "../MapObjects/Structure.h"
+#include "../StructureCatalogue.h"
+#include "../Common.h"
+#include "../StorableResources.h"
```

Comment:
Seeing as how these are newly added and not referenced in the header file, it would be better to move new includes to the implementation file instead. That would avoid increasing transitive includes, which affect compile times.

Reply:
To clear things up about this comment and the similar one on the other PR. Move needed includes (in this case it may not be needed) to the .cpp.